client/src/components/App.jsx

```diff
@@ -8,20 +8,24 @@ import Reviewsdata from './Reviews/reviewsData.js';
 import QuestionListdata from './Q&A/Q&AlistData.js';
 import relatedItemsdata from './RelatedItems/relatedItemsData.js';
 import productListDummyData  from './ProductDetails/sampledata/productListDummyData.js';
-import sampleThumbnails from './ProductDetails/sampleData/imageThumbnailsArray.js'
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      currentProductID: 39337,
+      currentProductID: '39333',
+      currentStyleID: 234004,
       productList: productListDummyData,
       reviews: [],
       questionList: QuestionListdata,
       relatedItems: relatedItemsdata,
-      imageGallery: sampleThumbnails
 
     };
+    this.changeCurrentStyle = this.changeCurrentStyle.bind(this);
```

Comment:
@beddo018 @RehReis Do either of you need access to the currentStyleID? I'm just wondering if it might be helpful to keep this logic within the ProductOverview directory

Reply:
Great idea. I will look into this refactor Next week